include/ua_config.h.in

```diff
@@ -29,6 +29,8 @@ extern "C" {
 #cmakedefine UA_ENABLE_MULTITHREADING
 #cmakedefine UA_ENABLE_PUBSUB
 #cmakedefine UA_ENABLE_ENCRYPTION
+#cmakedefine UA_ENABLE_EVENTS
```

Comment:
Can we rename to UA_ENABLE_SUBSCRIPTIONS_EVENTS?
Can we ensure that this is only enabled together with subscriptions in cmake?